Gordon360/Controllers/ScheduleControlController.cs

```diff
@@ -94,8 +95,8 @@ public ActionResult UpdateSchedulePrivacy(string value)
         /// <param name="value">New description</param>
         /// <returns></returns>
         [HttpPut]
-        [Route("description/{value}")]
-        public ActionResult UpdateDescription(string value)
+        [Route("description")]
+        public ActionResult UpdateDescription([FromBody]String value)
```

Comment:
```suggestion
        public ActionResult UpdateDescription([FromBody]string value)
```
Despite the fact that there is little difference, please be consistent with the rest of our routes 

`String` is a .NET object type
`string` is a C# alias for System.String